pyproject.toml

```
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of base32-lib
+# Copyright (C) 2019 CERN.
+# Copyright (C) 2019 Northwestern University,
+#                    Galter Health Sciences Library & Learning Center.
+# Copyright (C) 2025 Front Matter.
+
+# base32-lib is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+[build-system]
+requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.2"]
```

In other modernized project (3.9+), I just cranked it to 61+ and no setuptools_scm. What was the specific need for that one?

```suggestion
requires = ["setuptools>=61", "wheel"]
```

---

not needed